package-lock.json

```diff
@@ -1,26 +1,26 @@
 {
   "name": "@closeio/best-practices-documentation",
   "version": "0.4.2",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
different npm versions? should we add nvmrc to the repo?

Reply:
Do you think it's worth keeping the same `lockfileVersion`? There's indeed a way to keep that the same with `--lockfile-version`

Reply:
More info about it at https://docs.npmjs.com/cli/v8/using-npm/config#lockfile-version

Reply:
I see no issues with updating the lockfile version. But adding an .nvmrc is a great idea!

Reply:
Agreed. Is it fine to use the same as `close-ui` (`18.16.1`)? or should we go for the current LTS (`20.9.0`)?

Reply:
I'll merge it to address the security warning, and we can update the node version if needed in a separate PR.